### ISSUE TITLE
Build failure during test run

### ISSUE THREAD

**Issue by jens-schnufte** [IDPF/epubcheck#528]
This is a fresh clone of the repo, building on Linux:
```
JAVA_HOME=/opt/oracle-jdk-bin-1.8.0.40 mvn install
```
It looks like the order in that list has been jumbled around?

```
Tests run: 30, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 2.704 sec <<< FAILURE!
testDuplicateZipEntriesIssue265b(com.adobe.epubcheck.api.Epub30CheckTest)  Time elapsed: 0.042 sec  <<< FAILURE!
java.lang.AssertionError: The warning results do not match expected:<[MessageId.PKG_012, MessageId.OPF_061, MessageId.OPF_003, MessageId.PKG_012]> but was:<[MessageId.OPF_003, MessageId.PKG_012, MessageId.OPF_061, MessageId.PKG_012]>
        at org.junit.Assert.fail(Assert.java:93)
        at org.junit.Assert.failNotEquals(Assert.java:647)
        at org.junit.Assert.assertEquals(Assert.java:128)
        at com.adobe.epubcheck.api.AbstractEpubCheckTest.testValidateDocument(AbstractEpubCheckTest.java:127)
        at com.adobe.epubcheck.api.AbstractEpubCheckTest.testValidateDocument(AbstractEpubCheckTest.java:63)
        at com.adobe.epubcheck.api.AbstractEpubCheckTest.testValidateDocument(AbstractEpubCheckTest.java:52)
        at com.adobe.epubcheck.api.Epub30CheckTest.testDuplicateZipEntriesIssue265b(Epub30CheckTest.java:282)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:497)
        at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:45)
        at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
        at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:42)
        at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
        at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:263)
        at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:68)
        at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:47)
        at org.junit.runners.ParentRunner$3.run(ParentRunner.java:231)
        at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:60)
        at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:229)
        at org.junit.runners.ParentRunner.access$000(ParentRunner.java:50)
        at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:222)
        at org.junit.runners.ParentRunner.run(ParentRunner.java:300)
        at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:252)
        at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:141)
        at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:112)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:497)
        at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
        at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
        at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
        at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
        at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
```

**Comment by rdeltour** [comment]
This seems to be a duplicate of #364, s/b fixed in pending PR #521. Please reopen if it doesn't! Thx.

**Comment by jens-schnufte** [comment]
Yes, was a duplicate and the complete tree built with #521. Thanks!
